UI_UX/Frontend/src/components/Achievements.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Achievements.css';
+
+export class Achievements extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <ol className="achievements">
+        {this.props.achievements.map(milestone => (
+          <li key={milestone.endpoint}>
+            <div>
+              <b>{milestone.title}</b>
```

Comment:
Since we do BEM, we ideally can style our code without relying on the semantic structure of it. So no problem on using `b`, but might be better to add a class name to complement it.